vector-im/element-android
# [Location sharing] - Navigation to Map view from live location message (PSF-888)

## Type of change


## Content


Add possibility to navigate to a map screen from a running live location share message.
User pins on map and other UI will be implemented in another pull request.

## Motivation and context


Part of #6012 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/169058312-d49fb926-858a-4a36-a5cd-e599ff1ffc68.gif" width=300 />

## Tests



- Ensure to enable location sharing in Settings -> Preferences
- Ensure to enable developer mode in Settings -> Advanced
- Ensure to enable the feature flaf for live location sharing
- Start a live location share in a room
- Press the running live message
- Check a new screen is opened with a map of the world
- Press back
- Stop the live
- Check pressing on the ended live message does nothing

## Tested devices

- OS version(s): Android 11

## Checklist
